guides/source/rails_on_rack.md

```diff
@@ -215,9 +215,9 @@ Much of Action Controller's functionality is implemented as Middlewares. The fol
 
 * Used to serve static files from the public directory. Disabled if `config.public_file_server.enabled` is `false`.
 
-**`Rack::Lock`**
```

Comment:
ping @matthewd - you still think Rack::Lock should be in there even though it is explained in configuring.md?